frontend/src/metabase/timelines/collections/components/TimelineEmptyState/TimelineEmptyState.tsx

```diff
@@ -44,47 +26,34 @@ const TimelineEmptyState = ({
 
   const applicationName = useSelector(getApplicationName);
   return (
-    <EmptyStateRoot>
-      <EmptyStateBody>
-        <EmptyStateChart>
-          <svg width="231" height="128" xmlns="http://www.w3.org/2000/svg">
-            <path
-              fill="currentColor"
-              d="M230.142 1.766 118.697 95.283a5 5 0 0 1-8.877 4.461L.75 127.97l-.501-1.937 108.827-28.16a5 5 0 0 1 8.557-4.306L228.857.233l1.285 1.532ZM114 99.5a2.5 2.5 0 1 0 0-5 2.5 2.5 0 0 0 0 5Z"
-              fillRule="evenodd"
-              clipRule="evenodd"
-            />
-          </svg>
-        </EmptyStateChart>
-        <EmptyStateTooltip>
-          <EmptyStateTooltipIcon name="mail_filled" />
-          <EmptyStateTooltipBody>
-            <EmptyStateTooltipTitle>{t`Launch of v2.0`}</EmptyStateTooltipTitle>
-            <EmptyStateTooltipDate value={date} unit="day" data-server-date />
-          </EmptyStateTooltipBody>
-        </EmptyStateTooltip>
-        <EmptyStateThread>
-          <EmptyStateThreadLine />
-          <EmptyStateThreadIconContainer>
-            <EmptyStateThreadIcon name="star" />
-          </EmptyStateThreadIconContainer>
-          <EmptyStateThreadLine />
-        </EmptyStateThread>
-        <EmptyStateMessage>
+    <Stack align="center" ta="center" gap="lg">
+      <Tooltip label={t`Launch of v2.0`} offset={-24} opened>
```

Comment:
❤️

Reply:
I also like the change away from 'anomaly', which for me has a slightly negative feel

Reply:
Maybe 'Product launch' could also work - simple, resonant

Reply:
I had the same feeling re: "The anomaly". 
The reason I chose "Launch of v2.0" is that it exists already in translations.